DevGraphs.md

```diff
@@ -0,0 +1,48 @@
+# Preview: Dev Graphs
+
+We have wanted to make Studio's tools helpful to folks during local development for a long time. Up until now, Studio has largely been tailored to work with graphs already running in production.
+
+Dev Graphs are **a new type of graph** that users can create in Studio. They have special properties that have been optimized for development-time concerns.
+
+### Dev Graph Properties
```

Comment:
Could maybe shorten this list if you want -- 1 and 3 could be collapsed, 2 and 5 could be collapsed.